envoy/config/metrics/v2/stats.proto

```diff
@@ -155,6 +155,24 @@ message StatsdSink {
     // Envoy will connect to this cluster to flush statistics.
     string tcp_cluster_name = 2;
   }
+  // [#not-implemented-hide:] Optional custom prefix for StatsdSink. If
+  // specified, this will override the default prefix.
+  // For example:
+  // .. code-block:: json
+  //   prefix : "envoy-prod"
+  // will change emitted stats to
+  // .. code-block:: cpp
+  //   envoy-prod.test_counter:1|c
+  //   envoy-prod.test_timer:5|ms
+  //
+  // Note that the default prefix, "envoy", will be used if a prefix is not
+  // specified.
+  // Stats with default prefix:
+  // .. code-block:: cpp
+  //   envoy.test_counter:1|c
+  //   envoy.test_timer:5|ms
+
```

Comment:
Please view the rendered docs. This blank line makes the docs not attach.

Reply:
Sorry! This is what it looks like now: ![](https://i.imgur.com/liOhjaU.png)